### ISSUE TITLE
Container element is empty (IU Accessibility Mandate)

### ISSUE THREAD

**Issue by mdalmau** [IUBLibTech/newton_chymistry#133]
20 occurrences of this issue ... Looks like the HTML is spitting out an empty unordered list in the search / search results pages:
`<ul class="results"></ul>`

--------------------------
IU has a mandate that all web sites published on IU servers must meet WCAG I and II Accessibility requirements. If the requirements are NOT met, sites will be taken down. To learn more about this initiative, visit the [Digital Strategy Initiative](https://ucm.iu.edu/digital-strategy/index.html) web site.

- [ ] Ask Bill who should be added to the SiteImprove tool

**Comment by mdalmau** [comment]
@tubesoft : At the Newton meeting today, Bill said you would be working on these accessibility issues. I requested SiteImprove access for you. Once you have access, you will login using your @iu credentials. If you want to read more about SIteImprove and accessibility, LOGIN to the KB and [start here](https://kb.iu.edu/d/webs). 

Could you have a look at this issue and confirm it's something you can fix? Once you confirm, we can figure out a testing workflow.

**Comment by tubesoft** [comment]
In `solr-response-to-html.xsl`, the search results are printed between `<ul class="results">` and `</ul>`. That is, if the search results are none, only `<ul class="results"></ul>` remains (See image below). From the perspective of programming, it does not seem to be too bad. In order to omit `<ul class="results"></ul>`, I would need to add a condition of printing `<ul class="results"></ul>`.  Do we really need to follow this warning strictly? Program-wise, leaving it as it is is much simpler.

<img width="773" alt="image" src="https://github.com/IUBLibTech/newton_chymistry/assets/17519127/7c92a6b8-5796-401c-b32c-fb1409e475c5">

**Comment by mdalmau** [comment]
@tubesoft : yes, all these issues must be fixed. There’s no way around this. If you think it’s beyond your comfort level, we can set that aside for Bill to figure out with Conal but anything you can do, please do because we are under a deadline! Thank you.

**Comment by mdalmau** [comment]
Since this issue is now part of https://github.com/IUBLibTech/newton_chymistry/pull/146, I will mark this issue as "done" so I can better see the remaining accessibility issues!
